docs/source/features/sheet_management/schema.rst

```diff
@@ -4,3 +4,217 @@
 ########
  Schema
 ########
+
+Each sheet has a record type schema to describe the name and the type of each column.
+
+Graviti use `Portex`_ schema language to define the schema, please refer to its documentation for the grammar.
+
+.. _Portex: https://github.com/Project-OpenBytes/portex
+
+Graviti SDK supports the python interaction with the Portex schema.
+
+
+*****************
+ Primitive Types
+*****************
+
+Graviti SDK provides classes to initialize Portex primitive types.
+
+.. code:: python
+
+   import graviti.portex as pt
+   pt.boolean()
+   pt.binary()
+   pt.string()
+   pt.int32()
+   pt.int64()
+   pt.float32()
+   pt.float64()
+
+***************
+ Complex Types
+***************
+
+Graviti SDK provides classes to initialize Portex complex types.
+
+enum
+====
+
+An enum type can be created by giving the values.
+
+.. code:: python
+
+   import graviti.portex as pt
+   enum = pt.enum(values=["a", "b", "c"])
+   enum.values
+
+.. note::
+   The values of the enum type must be strings with the format ``[A-Za-z_][A-Za-z0-9_]*``.
+
+array
+=====
+
+The array type can be created by giving the item type and length.
+
+
+.. code:: python
+
+   import graviti.portex as pt
+   array = pt.array(pt.int32())
+   array = pt.array(pt.boolean(), length=2)
+   array.items
+   array.length
+
+tensor
+======
+
+The tensor type can be created by giving the item type and shape.
+
+.. code:: python
+
+   import graviti.portex as pt
+   tensor = pt.tensor((2, 2), pt.int32())
+   tensor.items
+   tensor.shape
+
+
+record
+======
+
+The record type can be created by giving the fields, including names and types.
+The record type is used to describe the name and type of each column of the tabular data,
+and all the primitive and complex types mentioned above can be used here for each column type.
+
+The names and types can be accessed by `fields`, which acts like a dict whose key is the column name and the value is column type.
+
+.. code:: python
+
+   import graviti.portex as pt
+   record = pt.record([("x", pt.int32()), ("y", pt.int32()), ("categories", pt.enum(values=["cat", "dog"]))])
+   record.fields
```

Comment:
Format it, make it easier to read. And do we need to add a example for using dict to initialize pt.record here?